src/controllers/util/index.ts

```diff
@@ -1,15 +1,19 @@
 import { Context } from 'koa';
 import { API_VERSION } from '../../routes/utils/constants';
-import { isHttpStatusSuccess } from '../../v0/util';
+import { getCompatibleStatusCode } from '../../adapters/utils/networkUtils';
 
 export default class ControllerUtility {
+  private static getCompatibleStatusCode(status: number): number {
```

Comment:
Is there a plan to use this API anywhere outside this class? Otherwise, we can remove this and use the utility directly.

Reply:
At a future we want to have `getCompatibleStatusCode` in this utility itself. We have only added this for supporting the javascript routes(`versionedRouter.js`)